CHANGELOG.md

```diff
@@ -20,7 +59,7 @@ Monitor the release status by regions at [AKS-Release-Tracker](https://releases.
 * Preview Features:
   * AKS version [1.30](https://kubernetes.io/blog/2024/04/17/kubernetes-v1-30-release/) is available in preview. 
```

Comment:
Why do we need to call this out.. The last release 15 days ago had the same line.

Reply:
this line hasn't changed. you are commenting on the previous release :)